tests/runner/remote_state_reads/init.json

```diff
@@ -7,7 +7,7 @@
     {
         "vname" : "_this_address",
         "type" : "ByStr20",
-        "value" : "0xabfeccdc9012345678901234567890f777564321"
+        "value" : "0x3620c286757a29985cee194eb90064270fb65414"
```

Comment:
Why the change of  `_this_address`?

Reply:
Because `_this_address` in the blockchain is set based on an actual account, and that's what is passed through `init.json` (the init.json in the test will have its `_this_address` value replaced). But this becomes a problem because we have `_this_address.SomeType` used. That will now fail. So it was simpler to actually set the `_this_address` to an account used in the blockchain test.

Reply:
That makes sense. Feel free to merge.